### DISCUSSION TITLE
Access ABCD Data

### DISCUSSION THREAD

**Discussion by Tzung-Shiang** [now-i-know-my-abcd/docs#93] [ABCD Updates]
Hi everyone, I’ve noticed that the message ‘Packaging, downloading data, and MINDAR databases are currently unavailable’ on the National Institute of Mental Health Data Archive website has been up for a couple of weeks now. Does anyone know of alternative methods to access or download ABCD data while this issue persists? Any suggestions or workarounds would be greatly appreciated. Thanks!

**Comment by angielaird** [comment]
Unfortunately, NDA is down and this affects all data in the archive, not just ABCD. It is unclear when it will be back up. 

The ABCD Study has posted an announcement about this issue on the [ABCD Wiki splash page](https://wiki.abcdstudy.org). That page also include an announcement about the current changes in data sharing policies due to a recent [executive order issued by the Biden Administration](https://www.whitehouse.gov/briefing-room/presidential-actions/2024/02/28/executive-order-on-preventing-access-to-americans-bulk-sensitive-personal-data-and-united-states-government-related-data-by-countries-of-concern/#%3A~%3Atext=%28c%29%20The%20term%20“country%2CStates%20or%20the%20security%20and).
